awx_collection/plugins/modules/tower_workflow_job_template_schema.py

```diff
@@ -0,0 +1,378 @@
+      description:
+        - Job tags applied as a prompt, if job template prompts for job tags
+      type: str
+    skip_tags:
+      description:
+        - Tags to skip, applied as a prompt, if job tempalte prompts for job tags
+      type: str
+    limit:
+      description:
+        - Limit to act on, applied as a prompt, if job template prompts for limit
+      type: str
+    diff_mode:
+      description:
+        - Run diff mode, applied as a prompt, if job template prompts for diff mode
+      type: bool
+    verbosity:
+      description:
+        - Verbosity applied as a prompt, if job template prompts for verbosity
+      type: str
+      choices:
+        - '0'
+        - '1'
+        - '2'
+        - '3'
+        - '4'
+        - '5'
+    unified_job_template:
+      description:
+        - Name of unified job template to run in the workflow.
+        - Can be a job template, project, inventory source, etc.
+        - Omit if creating an approval node (not yet implemented).
+      type: str
+    all_parents_must_converge:
+      description:
+        - If enabled then the node will only run if all of the parent nodes have met the criteria to reach this node
+      type: bool
+    identifier:
+      description:
+        - An identifier for this node that is unique within its workflow.
+        - It is copied to workflow job nodes corresponding to this node.
+      required: True
+      type: str
+    always_nodes:
+      description:
+        - Nodes that will run after this node completes.
+        - List of node identifiers.
+      type: list
+      elements: str
+    success_nodes:
+      description:
+        - Nodes that will run after this node on success.
+        - List of node identifiers.
+      type: list
+      elements: str
+    failure_nodes:
+      description:
+        - Nodes that will run after this node on failure.
+        - List of node identifiers.
+      type: list
+      elements: str
+    credentials:
+      description:
+        - Credentials to be applied to job as launch-time prompts.
+        - List of credential names.
+        - Uniqueness is not handled rigorously.
+      type: list
+      elements: str
+    state:
+      description:
+        - Desired state of the resource.
+      choices: ["present", "absent"]
+      default: "present"
+      type: str
+
+extends_documentation_fragment: awx.awx.auth
+'''
+
+EXAMPLES = '''
+- name: Create a node, follows tower_workflow_job_template example
+  tower_workflow_job_template_node:
+    identifier: my-first-node
+    workflow: example-workflow
+    unified_job_template: jt-for-node-use
+    organization: Default  # organization of workflow job template
+    extra_data:
+      foo_key: bar_value
+
+- name: Create parent node for prior node
+  tower_workflow_job_template_node:
+    identifier: my-root-node
+    workflow: example-workflow
+    unified_job_template: jt-for-node-use
+    organization: Default
+    success_nodes:
+      - my-first-node
```

Comment:
It would be good to have examples with schema to start setting basic expectations